graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/DgsWebfluxConfigurationProperties.kt

```diff
@@ -48,6 +48,7 @@ class DgsWebfluxConfigurationProperties(
      * Configuration properties for the GraphiQL endpoint.
      */
     data class DgsGraphiQLConfigurationProperties(
+        @DefaultValue("true") var enabled: Boolean = true,
```

Comment:
Where is this used?

Reply:
It's used by Spring's [metadata generator](https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html#appendix.configuration-metadata.annotation-processor). The property isn't used directly by DGS b/c it's referred to in full dot notation as a `@Conditional`. However, it's included in the WebMVC equivalent of this class for Spring configuration autocompletion; see:

https://github.com/Netflix/dgs-framework/blob/8fa2368bf5e5ffb1a35a34e10e4bc328e02945d2/graphql-dgs-spring-webflux-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webflux/autoconfiguration/DgsWebFluxAutoConfiguration.kt#L144-L147